tools/mongodb/utils/execute-script.sh

```diff
@@ -0,0 +1,38 @@
+#! /usr/bin/env bash
+
+DIR="$(realpath "$(dirname "${BASH_SOURCE[0]}")")"
+# Used to provide the correct environment to execute a MongoDB scripts. See `export-all.sh` in the parent directory for an example. You can also read the documentation for more information.
+if [ $# -eq 0 ]; then
+    echo "No arguments provided"
+    echo ""
+    echo "Usage: $0 <script-name>"
+    echo ""
+    echo "Available scripts:"
+    ls $DIR/../scripts/*.js | xargs -n 1 basename | sed 's/\.js//g' | sed 's/^/ - /g'
+    exit 1
+fi
+
+echo "Executing script: $1"
+
+# Get MongoDB Password
+MPASS=$("$DIR/mongodb-password.sh")
+# Get MongoDB Username
+MUSER=$("$DIR/mongodb-username.sh")
+# Get MongoDB IPS
+MONGO_IPS=$("$DIR/mongodb-ip.sh")
+
+# Get MongoDB Hosts (IP:PORT)
+MONGO_HOSTS=$(echo $MONGO_IPS | sed 's/ /:27017 /g'):27017
```

Comment:
What does it do if there are multiple IPs?

Reply:
Currently, there is multiple IPs. 

You would say more than 2? I works fine!

Here the input `<ip> <ip> <ip>` and the output `<ip>:27017 <ip>:27017 <ip>:27017`